es6-lib/decoders/layer.js

```diff
@@ -240,9 +241,16 @@ class Layer extends Duplex {
     return this._count;
   }
 
+  /**
+   * Returns the projection for the row at index
+   */
   _getProjectionFor(index) {
     if (!this._crsMap[index]) return this.defaultCrs;
-    return srs.parse(this._crsMap[index]);
+    var unparsed = this._crsMap[index];
+    if(!this._crsCache[unparsed]) {
+      this._crsCache[unparsed] = srs.parse(unparsed);
```

Comment:
Would it be possible to build this caching behavior into `srs.parse` in the upstream library (via a PR there or something)?

This seems like it was probably not intended behavior.